deploy/nginx.service.yml.tpl

```diff
@@ -1,23 +1,22 @@
-# This is the backend service
-apiVersion: v1
 kind: Service
+apiVersion: v1
 metadata:
-  name: nginx-ingress-lb
+  name: ingress-nginx
+  namespace: ingress-nginx
   labels:
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
 spec:
   externalTrafficPolicy: Local
   type: LoadBalancer
   loadBalancerIP: {{ STATIC_IP }}
-  ports:
-  - port: 80
-    name: http
-    targetPort: 80
-  - port: 443
-    name: https
-    targetPort: 443
   selector:
-    # Selects nginx-ingress-controller pods
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
+  ports:
```

Comment:
Similar question to the one above regarding the ports, is there a reason for moving the configurations down?

Reply:
no, it does not matter. I have copied this file from kubernetes.